cwr/parser.go

```diff
@@ -24,13 +24,17 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"os"
 	"os/exec"
 )
 
 // ParseCWRFile parse a give cwr file and returns an array of registeredWorks
 func ParseCWRFile(cwrFileReader io.Reader, CWRDataApiPath string) (registeredWorks []*RegisteredWork, err error) {
 	//phase 1 : Transform cwr formatted file to cwr-json file using CWR-DataApi python script.
 	cmd := exec.Command("python3", CWRDataApiPath+"/cwr2json.py")
+	cmd.Env = append(os.Environ(),
+		"PYTHONIOENCODING=ISO-8859-1",
```

Comment:
We should add a comment explaining why we need to use an explicit encoding.

Reply:
Add a comment at e32a4a1